rootfs/scheduler/states.py

```diff
@@ -1,7 +1,34 @@
-import enum
+from enum import Enum, unique
 
 
-class JobState(enum.Enum):
+class OrderedEnum(Enum):
+    def __ge__(self, other):
+        if self.__class__ is other.__class__:
+            return self.value >= other.value
+
+        return NotImplemented
+
+    def __gt__(self, other):
+        if self.__class__ is other.__class__:
+            return self.value > other.value
+
+        return NotImplemented
+
+    def __le__(self, other):
+        if self.__class__ is other.__class__:
+            return self.value <= other.value
+
+        return NotImplemented
+
+    def __lt__(self, other):
+        if self.__class__ is other.__class__:
+            return self.value < other.value
+
+        return NotImplemented
+
+
+@unique
```

Comment:
Cool, I didn't know about the [@unique Enum decorator](https://docs.python.org/3/library/enum.html#enum.unique).